src/pymovements/events/properties.py

```diff
@@ -143,6 +170,18 @@ def location(
 ) -> pl.Expr:
     """Location of an event.
 
+    For method ``mean`` the location is calculated as:
+
+    .. math::
+        \\text{Location} = \\frac{1}{n} \\sum_{i=1}^n \\text{position}_i
```

Comment:
I would say the first, because it not only assumes 2D. We might deal with 3D -- see #495. and several eyes (left, right, cyclop, ...). I don't even wanna know about VR.

Reply:
The first one is better